.github/workflows/scripts/swift-integration.dart

```diff
@@ -0,0 +1,105 @@
+// Copyright 2024, the Chromium project authors.  Please see the AUTHORS file
+// for details. All rights reserved. Use of this source code is governed by a
+// BSD-style license that can be found in the LICENSE file.
+
+import 'dart:io';
+import 'dart:convert';
+
+void main() async {
+  await buildSwiftExampleApp('ios', 'firebase_core');
+  await buildSwiftExampleApp('ios', 'cloud_firestore');
+  await buildSwiftExampleApp('macos', 'firebase_core');
+  await buildSwiftExampleApp('macos', 'cloud_firestore');
```

Comment:
Will we need to do the other packages? Or is it just because cloud_firestore has its own SPM package?